README.md

```diff
@@ -30,7 +30,8 @@ Projects listed on this page are open source that Microsoft maintain or contribu
+|                                      | [Envoy Proxy](https://www.envoyproxy.io/)                                                                                                                                                                                                                          | A high performance, cloud-native proxy                                                                                                                                                                                                                         | CNCF: graduated                                                                  | [Envoy Slack](https://envoyproxy.slack.com) <br> [GitHub issues](https://github.com/envoyproxy/envoy/issues)                                                                                                                                                                                                                                                                          |
+|                                      | [Service Mesh Interface (SMI) Spec](https://smi-spec.io/)                                                                                                                                                                                                                          | A standard interface for service meshes on Kubernetes                                                                                                                                                                                                                         | Deprecated; consider using [Gateway API](https://gateway-api.sigs.k8s.io/)                                                                 | [#smi](https://cloud-native.slack.com/messages/smi) <br> [GitHub issues](https://github.com/servicemeshinterface/smi-spec/issues)                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Can you pls move this one and all other deprecated stuff to the past project table below?

xref: https://github.com/Azure/container-upstream/pull/142/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R53

Reply:
Sorry for the delay! done

Reply:
Did you mean to remove this one since it's archived?

Reply:
Ah boo bad rebase